lib/Optimizer/CodeGen/Pipelines.cpp

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "cudaq/Optimizer/CodeGen/Pipelines.h"
+
+using namespace mlir;
+
+void cudaq::opt::commonPipelineConvertToQIR(
+    PassManager &pm, const std::optional<StringRef> &convertTo) {
```

Comment:
If you accept the above suggestion, you'd need this change, too.
```suggestion
    PassManager &pm, const std::optional<StringRef> convertTo) {
```